drivers/hmis_external_apis/app/jobs/hmis_external_apis/ac_hmis/update_unit_availability_job.rb

```diff
@@ -6,38 +6,127 @@
 
 module HmisExternalApis::AcHmis
   class UpdateUnitAvailabilityJob < ApplicationJob
+    queue_as ENV.fetch('DJ_SHORT_QUEUE_NAME', :short_running)
```

Comment:
Can this inherit from BaseJob instead of ApplicationJob.  That'll bring along the logic for restarting if it's on the wrong deployment.